README.md

```diff
@@ -31,6 +31,7 @@ To use LLMinator, follow these simple steps:
 |----------|----------|----------|
 | --host | 127.0.0.1 | Host or IP address on which the server will listen for incoming connections.|
```

Comment:
Consistency issue, sometime there is . available & somewhere not. 
Space is also missing sometimes. 

Please make sure to keep the consistency across you code.